src/Ethereum.sol

```diff
@@ -16,10 +16,11 @@ library Ethereum {
     /*** SparkDAO Addresses                                                                                         ***/
     /******************************************************************************************************************/
 
-    address internal constant SPARK_PROXY = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+    address internal constant META_MORPHO_VAULT = 0x73e65DBD630f90604062f6E02fAb9138e713edD9;
```

Comment:
I would label this as `MORPHO_VAULT_DAI_1`.

Reply:
Updated